.travis.yml

```diff
@@ -8,7 +8,6 @@ script:
 env:
   global:
     - DJANGO_SETTINGS_MODULE="meetuppizza.settings.ci"
-    - ADMIN_PASS=awesomepass
```

Comment:
Did you add this to the Travis project env settings instead? What's the reasoning behind keeping `DJANGO_SETTINGS_MODULE` here instead of doing the same?